konnectivity-client/pkg/client/client.go

```diff
@@ -134,6 +134,10 @@ type grpcTunnel struct {
 	// closing should only be accessed through atomic methods.
 	// TODO: switch this to an atomic.Bool once the client is exclusively buit with go1.19+
 	closing uint32
+
+	// Support a gauge tunnels metric by status. Access must be synchronized.
```

Comment:
Would we be better of replacing these two with https://pkg.go.dev/sync/atomic#Value ?
I think this obviates the need for writing a setMetric helper.

Reply:
Thanks, that's perfect here